### DISCUSSION TITLE
Invalidate cache for /cog/tile endpoint

### DISCUSSION THREAD

**Discussion by lawrencenika** [developmentseed/titiler#1010] [Q&A]
I was loading a url object at `/cog/tile?url=gs://abc/file.tif`, it works. But when I deleted `gs://abc/file.tif` and created a new `gs://abc/file.tif` with the different content, my titiler is still showing the old tile content? Is there a way to invalidate cache by an API call manually?

I m using GCS bucket without versioning.

**Comment by vincentsarago** [comment]
ref: https://github.com/developmentseed/titiler/discussions/816 https://github.com/rasterio/rasterio/issues/2181
